CancerGov/_src/StyleSheets/_tableSort.scss

```diff
@@ -55,62 +55,58 @@ table[data-sortable] td[data-sorted="true"]{
   table[data-sortable] {
     height: 100% !important;
   }
-  
-  table[data-sortable] tfoot tr td:last-child {
-    padding-bottom: 20px;
-    background-color: #ace;
-  }
 
-  table[data-sortable] thead tr th:first-child,
-  table[data-sortable] tbody tr td:first-child
-  /*,
-  table[data-sortable] tfoot tr td:first-child*/ {
-  //background: #ace;
-  top: auto;
-  left: 0.5px;
-  position: absolute !important;
-  width: 160px !important;
-  z-index: 3;
-  }
   table[data-sortable] thead tr th:first-child {
     //background-color: #ace;
     height: 7.85em;
-  }
-  table[data-sortable] tbody tr td:first-child {
-    background-color: #fff;
-    //left: -1px;
-    height: 200px !important;
-    line-height: 1.25em;
+    box-shadow: 0 2px 12px 3px rgba(0,0,0,.2);
   }
 
-  table[data-sortable] tbody tr td, 
-  /*table[data-sortable] tfoot tr td,*/ 
-  table[data-sortable] tr td {
-    height: 200px !important;
-    //padding: 20px;
-    //height: 100% !important;
+  table[data-sortable] thead tr th[data-sorted="false"] {
+    background-color: #2b7bba;
   }
-  table[data-sortable] tfoot tr td {
-    height: auto !important;
+  
+  table[data-sortable] thead tr th:first-child,
+  table[data-sortable] tbody tr td:first-child {
+    //background: #ace;
+    border-right: 1px solid #ccc;
+    border-left: 1.25px solid #ccc;
+    border-bottom: none;
+    top: auto;
+    left: 0px;
+    position: absolute !important;
```

Comment:
Are these `!important` declarations needed? Especially if we're scrubbing the cell width and height attributes from content items?

Reply:
We are removing the cell height attributes, not the width attributes at mobile.  The 160px width was a requested attribute.